lib/src/cmaf/utils/dash/DashManifest.ts

```diff
@@ -0,0 +1,64 @@
+type SegmentMpd = {
+	$: {
+		timescale?: number,
+		indexRangeExact?: boolean,
+		indexRange: string | null,
+	},
+	Initialization: [
+		{
+			$: {
+				range: string | null,
+			}
+		}
+	]
+}
+
+type Representation = {
+	$: {
+		id: string,
+		bandwidth: number,
+	},
+	BaseURL?: string[],
```

Comment:
Is the `BaseURL` an actual array? Or is it this way to consider content-steering in the future?
I wonder if it shouldn't be just a string, but if it has content-steering then we would have an array of `BaseURL`s
Might need to think about this a bit more